frontend/tests/e2e/features/logout.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Logout
+    As an admin
+    I want to log out of the system
+
+    Scenario: Successful logout
+        Given the user has browsed to the home page
```

Comment:
```suggestion
        Given the user has logged in with username "admin" and password "admin"
```
We can use this step to log in, the assertion of whether the user browsed to the home page or not will be done in this given step.

Reply:
I have the same suggestion